### PR TITLE
Eng 944 update discoverroute to show only base

### PR DESCRIPTION
**Pull request by eunice-chan**
Before, the Discover route (and Integration Details Page) shows all tables in the integration. Now, it only shows the base tables--user-created tables (loadspec) are filtered out.

### CODE REVIEW

File: src/golang/lib/workflow/operator/connector/load_params.go [hunk 1]
```diff
@@ -39,6 +39,50 @@ type S3LoadParams struct {
 	Format   string `json:"format"`
 }
 
+func CastToRelationalDBLoadParams(params LoadParams) (*RelationalDBLoadParams, bool) {
```
Comment (cw75) [inline src/golang/lib/workflow/operator/connector/load_params.go:42]: If all we care about is whether the load param is a relational db load param, can we just cast it to `RelationalDBLoadParams` instead of trying to cast to the specific ones?

File: src/ui/common/package-lock.json [hunk 2]
```diff
@@ -1,12 +1,12 @@
 {
   "name": "@aqueducthq/common",
-  "version": "0.0.6",
+  "version": "0.0.7",
```
Comment (cw75) [inline src/ui/common/package-lock.json:3]: Is this change intentional? Seem not relevant to this PR.

**Review by cw75 — COMMENTED** [review]
LGTM, a couple questions inline.

File: src/golang/lib/workflow/operator/connector/load_params.go (hunk 1 above)
Comment (likawind) [reply src/golang/lib/workflow/operator/connector/load_params.go:42]: Is this comment out-dated? It seems that this is exactly what the func is doing

**Review by likawind — APPROVED** [review]
The change LGTM!

@cw75 and @vsreekanti one question for you guys is what do we show if the user choose to append to an existing 'base' table? The current behavior will filter that table out, but I don't have good idea on what's a better way though

File: src/golang/lib/workflow/operator/connector/load_params.go (hunk 1 above)
Comment (eunice-chan) [reply src/golang/lib/workflow/operator/connector/load_params.go:42]: I can't do that directly because there is no generic RelationalDBLoadParamsWrapper and there was some weirdness when I was trying to make it a LoadParam. For now, I've decided to do something similar to https://github.com/aqueducthq/aqueduct/blob/377cd4d603b8160d1c647bb1149b63f8b3c0039c/src/golang/lib/workflow/operator/connector/extract_params.go#L70

**Comment by eunice-chan** [comment]
New:
Only CSV-uploaded & existing base tables.
image
Old:
The above plus the data tables saved by the user via `(table_artifact).save`
image
